src/phone-input/types.js

```diff
@@ -40,95 +37,102 @@ export type StateReducerT = (
   'aria-label': ?string,
-  /** Current input text value. Note, this should include the dial code of the selected country. */
-  text: string,
-  /** Current selected country value. Note, this expects an entire CountryT object. */
+  /** Sets aria-labelledby attribute of the input element. */
+  'aria-labelledby': ?string,
+  /** Sets aria-describedby attribute of the input element. */
+  'aria-describedby': ?string,
+  /** Defines the value of the country select. */
   country: CountryT,
-  /** Change handler of the text input to be called when a value is changed. */
-  onTextChange: (event: SyntheticInputEvent<HTMLInputElement>) => mixed,
-  /** Change handler of the country select to be called when a value is changed. */
-  onCountryChange: (event: OnChangeParamsT) => mixed,
-  /** A function that can be used to map iso codes to localized country names */
+  /** Defines if the component is disabled. */
+  disabled: boolean,
+  /** Renders component in 'error' state. */
+  error: boolean,
+  /** Sets the id attribute of the input element. */
+  id: ?string,
+  /** Sets the max height of the country select dropdown. */
+  maxDropdownHeight: string,
+  /** Sets the max width of the country select dropdown. */
+  maxDropdownWidth: string,
+  /** Function for mapping ISO codes to country names. Useful for localization of the country select dropdown. */
   mapIsoToLabel?: mapIsoToLabelT,
-  /** Defines the size of the text input. */
-  size?: SizeT,
-  /** Renders component in 'disabled' state. */
-  disabled?: boolean,
-  /** Defines a maximum dropdown height. The edge of the viewport will shrink the dropdown accordingly. */
-  maxDropdownHeight?: string,
-  /** Defines a maximum dropdown width. The edge of the viewport will shrink the dropdown accordingly. */
-  maxDropdownWidth?: string,
-  overrides?: OverridesT,
+  /** Sets the name attribute of the input element. */
+  name: ?string,
+  /** A handler for the country select's change events. */
+  onCountryChange: (event: OnChangeParamsT) => mixed,
+  /** A handler for the input element's change events. */
+  onTextChange: (event: SyntheticInputEvent<HTMLInputElement>) => mixed,
+  overrides: {
+    Input?: OverrideT<*>,
+    CountrySelectDropdown?: OverrideT<*>,
+    CountrySelectDropdownListItem?: OverrideT<*>,
+    CountrySelectDropdownFlagColumn?: OverrideT<*>,
+    CountrySelectDropdownNameColumn?: OverrideT<*>,
+    CountrySelectDropdownDialcodeColumn?: OverrideT<*>,
+    DialCode?: OverrideT<*>,
+    CountrySelect?: OverrideT<*>,
+  },
+  /** Sets the placeholder text for the input element.  */
+  placeholder?: string,
+  /** Renders component in 'positive' state. */
+  positive: boolean,
+  /** Sets the size of the component. */
+  size: SizeT,
+  /** Defines the value of the input element. */
+  text: string,
 };
 
-export type StatefulPhoneInputContainerPropsT = {
-  /** Sets aria-label attribute. */
-  'aria-label': ?string,
+export type StatefulContainerPropsT = {
```

Comment:
may want to revert the naming changes since consumers may have been importing them?